packages/@o3r/core/schematics/ng-add-create/templates/schematics/ng-add/index.ts

```diff
@@ -0,0 +1,39 @@
+import { chain, noop, Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+import type { NgAddSchematicsSchema } from './schema';
+import * as path from 'node:path';
+
+const doCustomAction = (tree: Tree, _context: SchematicContext) => {
+  // your custom code here
+  return tree;
+};
+
+/**
+ * Add Otter MyModule to an Otter Project
+ *
+ * @param options
+ */
+export function ngAdd(options: NgAddSchematicsSchema): Rule {
+  return async (_tree: Tree, context: SchematicContext) => {
+    try {
+      // use dynamic import to properly raise an exception if it is not an Otter project.
+      const { applyEsLintFix, install, ngAddPackages, getO3rPeerDeps } = await import('@o3r/schematics');
+      // retrieve dependencies following the /^@o3r\/.*/ pattern within the peerDependencies of the current module
+      const depsInfo = getO3rPeerDeps(path.resolve(__dirname, 'path', 'to', 'package.json'));
```

Comment:
The path could be actually  determined by the generator

Reply:
updated.
due to the fact that we control the packaging of the `schematics` folder, the `path` of `package.json` relative to this folder will be always the same.